From b7d282c97c13cc8ca6d42abffe2581a5f5bd76da Mon Sep 17 00:00:00 2001
From: Dominic Hargreaves <dom@earth.li>
Date: Tue, 20 Dec 2011 23:26:04 +0000
Subject: cpan/CPANPLUS: add NAME headings in modules with POD

This fixes the Debian Lintian warning about missing NAME sections in
manpages.

Bug: https://rt.cpan.org/Ticket/Display.html?id=73398
Bug-Debian: http://bugs.debian.org/650450

Patch-Name: fixes/manpage_name_CPANPLUS.diff
---
 cpan/CPANPLUS/lib/CPANPLUS/Backend.pm                            |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Backend/RV.pm                         |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Config.pm                             |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Config/System.pm                      |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Configure.pm                          |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Dist.pm                               |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Dist/Autobundle.pm                    |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Dist/MM.pm                            |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Error.pm                              |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/FAQ.pod                               |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Hacking.pod                           |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Internals.pm                          |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Internals/Extract.pm                  |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Internals/Fetch.pm                    |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Internals/Report.pm                   |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Internals/Search.pm                   |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Internals/Source.pm                   |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm                    |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Module.pm                             |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Module/Author.pm                      |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Module/Author/Fake.pm                 |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Module/Checksums.pm                   |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Module/Fake.pm                        |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Selfupdate.pm                         |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Shell.pm                              |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Shell/Default.pm                      |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/CustomSource.pm |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/Remote.pm       |    2 +-
 cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/Source.pm       |    2 +-
 29 files changed, 29 insertions(+), 29 deletions(-)

diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Backend.pm b/cpan/CPANPLUS/lib/CPANPLUS/Backend.pm
index 43e73b7..410dd3b 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Backend.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Backend.pm
@@ -36,7 +36,7 @@ $ENV{'PERL5_CPANPLUS_IS_VERSION'} = __PACKAGE__->VERSION;
 
 =head1 NAME
 
-CPANPLUS::Backend
+CPANPLUS::Backend - programmer's interface to CPANPLUS
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Backend/RV.pm b/cpan/CPANPLUS/lib/CPANPLUS/Backend/RV.pm
index e861a09..e329b56 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Backend/RV.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Backend/RV.pm
@@ -19,7 +19,7 @@ local $Params::Check::VERBOSE = 1;
 
 =head1 NAME
 
-CPANPLUS::Backend::RV
+CPANPLUS::Backend::RV - return value objects
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Config.pm b/cpan/CPANPLUS/lib/CPANPLUS/Config.pm
index 5cf11d3..3254ac7 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Config.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Config.pm
@@ -24,7 +24,7 @@ use Module::Load::Conditional   qw[check_install];
 
 =head1 NAME
 
-CPANPLUS::Config
+CPANPLUS::Config - configuration defaults and heuristics for CPANPLUS
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Config/System.pm b/cpan/CPANPLUS/lib/CPANPLUS/Config/System.pm
index 5e6e11e..4e9d8f3 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Config/System.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Config/System.pm
@@ -3,7 +3,7 @@
 
 =head1 NAME
 
-CPANPLUS::Config::System
+CPANPLUS::Config::System - system configuration for CPANPLUS
 
 =head1 DESCRIPTION
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Configure.pm b/cpan/CPANPLUS/lib/CPANPLUS/Configure.pm
index 25cbe5f..0409e39 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Configure.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Configure.pm
@@ -39,7 +39,7 @@ for my $meth ( qw[conf _lib _perl5lib]) {
 
 =head1 NAME
 
-CPANPLUS::Configure
+CPANPLUS::Configure - configuration for CPANPLUS
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Dist.pm b/cpan/CPANPLUS/lib/CPANPLUS/Dist.pm
index 20c74fc..67fc512 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Dist.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Dist.pm
@@ -22,7 +22,7 @@ local $Params::Check::VERBOSE = 1;
 
 =head1 NAME
 
-CPANPLUS::Dist
+CPANPLUS::Dist - base class for plugins
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Dist/Autobundle.pm b/cpan/CPANPLUS/lib/CPANPLUS/Dist/Autobundle.pm
index 9561dd9..6243fbd 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Dist/Autobundle.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Dist/Autobundle.pm
@@ -10,7 +10,7 @@ use base qw[CPANPLUS::Dist::Base];
 
 =head1 NAME
 
-CPANPLUS::Dist::Autobundle
+CPANPLUS::Dist::Autobundle - distribution class for installation snapshots
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Dist/MM.pm b/cpan/CPANPLUS/lib/CPANPLUS/Dist/MM.pm
index b2205e4..e44703f 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Dist/MM.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Dist/MM.pm
@@ -23,7 +23,7 @@ local $Params::Check::VERBOSE = 1;
 
 =head1 NAME
 
-CPANPLUS::Dist::MM
+CPANPLUS::Dist::MM - distribution class for MakeMaker related modules
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Error.pm b/cpan/CPANPLUS/lib/CPANPLUS/Error.pm
index c42435f..2625a98 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Error.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Error.pm
@@ -8,7 +8,7 @@ use Log::Message private => 0;;
 
 =head1 NAME
 
-CPANPLUS::Error
+CPANPLUS::Error - error handling for CPANPLUS
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/FAQ.pod b/cpan/CPANPLUS/lib/CPANPLUS/FAQ.pod
index 82bb57a..a62d457 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/FAQ.pod
+++ b/cpan/CPANPLUS/lib/CPANPLUS/FAQ.pod
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-CPANPLUS::FAQ
+CPANPLUS::FAQ - CPANPLUS Frequently Asked Questions
 
 =head1 DESCRIPTION
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Hacking.pod b/cpan/CPANPLUS/lib/CPANPLUS/Hacking.pod
index 6a754fb..9882a7b 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Hacking.pod
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Hacking.pod
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-CPANPLUS::Hacking
+CPANPLUS::Hacking - developing CPANPLUS
 
 =head1 DESCRIPTION
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Internals.pm b/cpan/CPANPLUS/lib/CPANPLUS/Internals.pm
index 4756421..a1c3eea 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Internals.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Internals.pm
@@ -48,7 +48,7 @@ $VERSION = "0.9103";
 
 =head1 NAME
 
-CPANPLUS::Internals
+CPANPLUS::Internals - CPANPLUS internals
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Extract.pm b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Extract.pm
index f30911b..c2de0c4 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Extract.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Extract.pm
@@ -19,7 +19,7 @@ local $Params::Check::VERBOSE = 1;
 
 =head1 NAME
 
-CPANPLUS::Internals::Extract
+CPANPLUS::Internals::Extract - internals for archive extraction
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Fetch.pm b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Fetch.pm
index 7dcb3c3..421296e 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Fetch.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Fetch.pm
@@ -19,7 +19,7 @@ $Params::Check::VERBOSE = 1;
 
 =head1 NAME
 
-CPANPLUS::Internals::Fetch
+CPANPLUS::Internals::Fetch - internals for fetching files
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Report.pm b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Report.pm
index 73a2d29..e551d69 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Report.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Report.pm
@@ -20,7 +20,7 @@ require CPANPLUS::Internals;
 
 =head1 NAME
 
-CPANPLUS::Internals::Report
+CPANPLUS::Internals::Report - internals for sending test reports
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Search.pm b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Search.pm
index a93626b..bc1d68b 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Search.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Search.pm
@@ -19,7 +19,7 @@ $Params::Check::VERBOSE = 1;
 
 =head1 NAME
 
-CPANPLUS::Internals::Search
+CPANPLUS::Internals::Search - internals for searching for modules
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Source.pm b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Source.pm
index 9637004..833aead 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Source.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Source.pm
@@ -74,7 +74,7 @@ $Params::Check::VERBOSE = 1;
 
 =head1 NAME
 
-CPANPLUS::Internals::Source
+CPANPLUS::Internals::Source - internals for updating source files
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm
index 31abaff..9bd5ff1 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm
@@ -17,7 +17,7 @@ local $Params::Check::VERBOSE = 1;
 
 =head1 NAME
 
-CPANPLUS::Internals::Utils
+CPANPLUS::Internals::Utils - convenience functions for CPANPLUS
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Module.pm b/cpan/CPANPLUS/lib/CPANPLUS/Module.pm
index 8d237ef..9a1ea72 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Module.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Module.pm
@@ -27,7 +27,7 @@ $Params::Check::VERBOSE = 1;
 
 =head1 NAME
 
-CPANPLUS::Module
+CPANPLUS::Module - CPAN module objects for CPANPLUS
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Module/Author.pm b/cpan/CPANPLUS/lib/CPANPLUS/Module/Author.pm
index 0daac67..a3ca18d 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Module/Author.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Module/Author.pm
@@ -13,7 +13,7 @@ local $Params::Check::VERBOSE = 1;
 
 =head1 NAME
 
-CPANPLUS::Module::Author
+CPANPLUS::Module::Author - CPAN author object for CPANPLUS
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Module/Author/Fake.pm b/cpan/CPANPLUS/lib/CPANPLUS/Module/Author/Fake.pm
index 3156c5c..46ec1ff 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Module/Author/Fake.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Module/Author/Fake.pm
@@ -17,7 +17,7 @@ $Params::Check::VERBOSE = 1;
 
 =head1 NAME
 
-CPANPLUS::Module::Author::Fake
+CPANPLUS::Module::Author::Fake - dummy author object for CPANPLUS
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Module/Checksums.pm b/cpan/CPANPLUS/lib/CPANPLUS/Module/Checksums.pm
index 3f9561e..3ebe690 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Module/Checksums.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Module/Checksums.pm
@@ -19,7 +19,7 @@ $Params::Check::VERBOSE = 1;
 
 =head1 NAME
 
-CPANPLUS::Module::Checksums
+CPANPLUS::Module::Checksums - checking the checksum of a distribution
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Module/Fake.pm b/cpan/CPANPLUS/lib/CPANPLUS/Module/Fake.pm
index 84d0233..d275020 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Module/Fake.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Module/Fake.pm
@@ -17,7 +17,7 @@ $Params::Check::VERBOSE = 1;
 
 =head1 NAME
 
-CPANPLUS::Module::Fake
+CPANPLUS::Module::Fake - fake module object for internal use
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Selfupdate.pm b/cpan/CPANPLUS/lib/CPANPLUS/Selfupdate.pm
index 6f2015c..c64e97a 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Selfupdate.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Selfupdate.pm
@@ -13,7 +13,7 @@ $Params::Check::VERBOSE = 1;
 
 =head1 NAME
 
-CPANPLUS::Selfupdate
+CPANPLUS::Selfupdate - self-updating for CPANPLUS
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Shell.pm b/cpan/CPANPLUS/lib/CPANPLUS/Shell.pm
index e3eb181..ddda561 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Shell.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Shell.pm
@@ -20,7 +20,7 @@ $DEFAULT    = SHELL_DEFAULT;
 
 =head1 NAME
 
-CPANPLUS::Shell
+CPANPLUS::Shell - base class for CPANPLUS shells
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default.pm b/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default.pm
index 35c02af..6544007 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default.pm
@@ -73,7 +73,7 @@ my $Prompt  = $Brand . '> ';
 
 =head1 NAME
 
-CPANPLUS::Shell::Default
+CPANPLUS::Shell::Default - the default CPANPLUS shell
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/CustomSource.pm b/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/CustomSource.pm
index 7b54681..94fed89 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/CustomSource.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/CustomSource.pm
@@ -9,7 +9,7 @@ use Locale::Maketext::Simple        Class => 'CPANPLUS', Style => 'gettext';
 
 =head1 NAME
 
-CPANPLUS::Shell::Default::Plugins::CustomSource 
+CPANPLUS::Shell::Default::Plugins::CustomSource - add custom sources to CPANPLUS
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/Remote.pm b/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/Remote.pm
index d2b829a..af4d66e 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/Remote.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/Remote.pm
@@ -9,7 +9,7 @@ use Locale::Maketext::Simple    Class => 'CPANPLUS', Style => 'gettext';
 
 =head1 NAME
 
-CPANPLUS::Shell::Default::Plugins::Remote
+CPANPLUS::Shell::Default::Plugins::Remote - connect to a remote CPANPLUS
 
 =head1 SYNOPSIS
 
diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/Source.pm b/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/Source.pm
index 889b3d3..82980ee 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/Source.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Shell/Default/Plugins/Source.pm
@@ -6,7 +6,7 @@ use Locale::Maketext::Simple    Class => 'CPANPLUS', Style => 'gettext';
 
 =head1 NAME
 
-CPANPLUS::Shell::Default::Plugins::Source 
+CPANPLUS::Shell::Default::Plugins::Source  - read in CPANPLUS commands
 
 =head1 SYNOPSIS
 
